drivers/spi/spi_nrfx_spis.c

```diff
@@ -105,42 +103,36 @@ static int configure(const struct device *dev,
 	return 0;
 }
 
-static void prepare_for_transfer(const struct device *dev)
+static void prepare_for_transfer(const struct device *dev,
+				 const uint8_t *tx_buf, size_t tx_buf_len,
+				 uint8_t *rx_buf, size_t rx_buf_len)
 {
 	struct spi_nrfx_data *dev_data = get_dev_data(dev);
 	const struct spi_nrfx_config *dev_config = get_dev_config(dev);
-	struct spi_context *ctx = &dev_data->ctx;
 	int status;
 
-	size_t buf_len = spi_context_max_continuous_chunk(ctx);
-
-	if (buf_len > 0) {
+	if (tx_buf_len > dev_config->max_buf_len ||
+	    rx_buf_len > dev_config->max_buf_len) {
+		LOG_ERR("Invalid buffer sizes: Tx %d/Rx %d",
+			tx_buf_len, rx_buf_len);
```

Comment:
I think that we can use the 100 char per line limit here and the lines would like better without breaking expressions.

Reply:
This file was written with the 80-characters limit, so a few longer lines added to it now will not look good. I don't want to introduce such inconsistency if it is not necessary.